scene/animation/tween.cpp

```diff
@@ -1716,7 +1716,7 @@ bool Tween::targeting_method(Object *p_object, StringName p_method, Object *p_in
 	Variant initial_val = p_initial->call(p_initial_method, NULL, 0, error);
 	ERR_FAIL_COND_V(error.error != Callable::CallError::CALL_OK, false);
 
-	// Convert initial INT values to REAL as they aer better for interpolation
+	// Convert initial INT values to FLOAT as they aer better for interpolation
```

Comment:
Saw this in the review "aer" typo.

I'll make a pr for this specific change.

Reply:
I saw it but i see no reason to leave it out of the replace